packages/resourceful-redux/test/unit/utils/get-resources.js

```diff
@@ -43,20 +43,20 @@ describe('getStatus', function() {
     });
   });
 
-  describe('labels', () => {
-    it('should return an empty array when the label does not exist', () => {
+  describe('requests', () => {
+    it('should return an empty array when the request does not exist', () => {
       expect(getResources(this.state, 'books', 'sandwiches')).to.deep.equal([]);
     });
 
-    it('should return an empty array when the label exists, and has no IDs', () => {
+    it('should return an empty array when the request exists, and has no IDs', () => {
```

Comment:
To differentiate between the generic concept of a "request" (which could be an HTTP request, for instance), and a request _name_, I would prefer that we use the full term, "request name," when referencing the `request` attribute on an action.

So tl;dr, for these test descriptions "label" should be replaced with "request name" rather than just "request" to prevent confusion